PROJECT

```diff
@@ -34,4 +35,20 @@ resources:
   kind: PulpRestore
   path: github.com/pulp/pulp-operator/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: pulpproject.org
+  group: repo-manager.pulpproject.org
+  kind: Pulp
+  path: github.com/pulp/pulp-operator/apis/repo-manager.pulpproject.org/v1beta1
```

Comment:
I believe you need to update the paths above for v1alpha1

Reply:
Nice catch!! Thank you!